src/camera.cpp

```diff
@@ -2,7 +2,6 @@
 
 acquisition::Camera::~Camera() {
 
-    pCam_ = NULL;
```

Comment:
We need to clear this pointer, otherwise the code spits out an error trace like this on exiting the node:
`7f3381926000-7f3381b25000 ---p 0021f000 08:06 13378414                   /usr/lib/x86_64-linux-gnu/libvpx.so.3.0.0
7f3381b25000-7f3381b27000 r--p 0021e000 08:06 13378414                   /usr/lib/x86_64-linux-gnu/libvpx.so.3.0.0
7f3381b27000-7f3381b28000 rw-p 00220000 08:06 13378414                   /usr/lib/x86_64-linux-gnu/libvpx.so.3.0.0
7f3381b28000-7f3381b2b000 rw-p 00000000 00:00 0   
...
`

Reply:
I reverted this change. This is a very strange (c-style) way to handle a class object but it's their API and we can't really change it so it is what it is.